### PR TITLE
Improved UserPage, closed modal after clicking on 'Додати' button on 'Дійсне членство' page #1502, another bugs/improvements 

### PR DESCRIPTION
**Pull request by christykobyletska**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/KadraVykhovnykiv/KadrasPage.tsx [hunk 1]
```diff
@@ -95,7 +96,12 @@ export const KadrasTable = () => {
                 ) : (
                 <></>
                 )}
-                <Input placeholder="Пошук" onChange={handleSearch} allowClear />
+                <Search
+                enterButton
```
Comment (OleksandrBilan) [inline src/pages/KadraVykhovnykiv/KadrasPage.tsx:100]: maybe some tabulation for this props?

File: src/pages/Precaution/PrecautionTable/PrecautionTable.tsx [hunk 2]
```diff
@@ -230,14 +230,14 @@ const PrecautionTable = () => {
...
             ) : (
               <></>
             )}
             <Search
                 enterButton
                 placeholder="Пошук"
+                allowClear
                 onChange={handleSearchChange}
                 onSearch={handleSearch}                
                />
```
Comment (OleksandrBilan) [inline src/pages/Precaution/PrecautionTable/PrecautionTable.tsx:243]: closing tag should have the same tabulation as the opening one

File: src/pages/userPage/Approvers/Approvers.tsx [hunk 3]
```diff
@@ -360,6 +372,7 @@ const Assignments = () => {
               </div>
             )
           )}
+            </div>
```
Comment (OleksandrBilan) [inline src/pages/userPage/Approvers/Approvers.tsx:375]: tabulation?

File: src/pages/userPage/Blanks/Blanks.tsx [hunk 4]
```diff
@@ -137,21 +137,31 @@ export const Blanks = () => {
...
-    return (!loading ? (
-        <Spinner />
-    ) : (
+    return loading === false ? (
+    <div className="kadraWrapper">
+        <Skeleton.Avatar
+        size={220}
+        active={true}
+        shape="circle"
+        className="img"
```
Comment (OleksandrBilan) [inline src/pages/userPage/Blanks/Blanks.tsx:146]: tabulate props?

File: src/pages/Precaution/PrecautionTable/PrecautionTable.tsx (hunk 2 above)
Comment (christykobyletska) [reply src/pages/Precaution/PrecautionTable/PrecautionTable.tsx]: Done

File: src/pages/userPage/Approvers/Approvers.tsx (hunk 3 above)
Comment (christykobyletska) [reply src/pages/userPage/Approvers/Approvers.tsx]: Done

File: src/pages/userPage/Blanks/Blanks.tsx (hunk 4 above)
Comment (christykobyletska) [reply src/pages/userPage/Blanks/Blanks.tsx]: Done

File: src/pages/KadraVykhovnykiv/KadrasPage.tsx (hunk 1 above)
Comment (christykobyletska) [reply src/pages/KadraVykhovnykiv/KadrasPage.tsx]: Done

**Review by OleksandrBilan — APPROVED** [review]

**Review by sharix4 — APPROVED** [review]

File: src/config.ts [hunk 5]
```diff
@@ -1,2 +1,2 @@
-const BASE_URL = "https://localhost:44350/api/";
+const BASE_URL = "https://localhost:5000/api/";
```
[the remark this answers is not in the document]
Comment (christykobyletska) [reply src/config.ts]: changed it already

**Review by andrii52 — APPROVED** [review]

**Pull request merged**